2-mandatory/4-people-i-know.js

```diff
@@ -393,24 +400,44 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
+
 let friendsWhoAreColleaguesOfStacie = [];
```

Comment:
Accessing various properties in an object is not always easy, neither to iterate through the object and logging diffirent properties and values based on a condition. These are all the foundation to good programming. Continue to practise and build skills.